security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -0,0 +1,44 @@
+//OpenShift File Integrity Operator Release Notes
+[id="file-integrity-operator-release-notes"]
+= File Integrity Operator release notes
+:context: file-integrity-operator-release-notes-v0
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+The File Integrity Operator for {product-title} deploys file integrity checking for RHCOS nodes.
+
+These release notes track the development of the File Integrity Operator in the {product-title}.
+
+For an overview of the File Integrity Operator, see xref:../../security/file_integrity_operator/file-integrity-operator-understanding.adoc#understanding-file-integrity-operator[Understanding the File Integrity Operator].
+
+[id="file-integrity-operator-release-notes-0-1-21"]
+== OpenShift File Integrity Operator 0.1.21
+
+The following advisory is available for the OpenShift File Integrity Operator 0.1.21:
+
+* link:https://access.redhat.com/errata/RHBA-2021:4631[RHBA-2021:4631 OpenShift File Integrity Operator Bug Fix and Enhancement Update]
+
+[id="file-integrity-operator-0-1-21-new-features-and-enhancements"]
+=== New features and enhancements
+
+* There are now informational metrics related to the ``FileIntegrity` scan results and processing. The metrics displayed on the monitoring dashboard on the console are labeled with the prefix of `file_integrity_operator_`.
++
+* There is a default `PrometheusRule` provided in the operator namespace that alerts with a warning if a node has an integrity failure for more than 1 second.
```

Comment:
```suggestion
* If a node has an integrity failure for more than 1 second, the default `PrometheusRule` provided in the operator namespace alerts with a warning.
```

Avoid using "there are", "there is" as a verb. This is part of the IBM Style Guide:

> there is, there are (verb)
> Avoid at the beginning of an independent clause. In most cases rewrite to remove the phrase. For example, instead of "There are three recovery control data sets in the product" write "The product includes three recovery control data sets".